internal/bminventory/inventory_test.go

```diff
@@ -5020,7 +5020,7 @@ var _ = Describe("cluster", func() {
 					validateNetworkConfiguration(actual.Payload, &clusterNetworks, &serviceNetworks, &machineNetworks)
 				})
 
-				It("Empty networks - valid", func() {
+				It("Empty networks (new API) - valid", func() {
```

Comment:
```suggestion
				It("Empty networks dual-stack - valid", func() {
```

Maybe we should just call it `v2` (even if it's not under the v2 path)